streams/quickstart/java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -30,7 +30,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <kafka.version>8.0.0-0-ccs</kafka.version>
```

Comment:
Conflict was in this line. Kept `8.0.0-0-ccs` -- while we already have a `4.0` branch in `confluentinc/kafka`, it seems `master` is still on `8.0` and was not bumped to `8.1`, yet.